modules/statics/src/coins.ts

```diff
@@ -135,6 +135,7 @@ export const coins = CoinMap.fromCoins([
   erc20('crpt', 'Crypterium', 18, '0x80a7e048f37a50500351c204cb407766fa3bae7f', UnderlyingAsset.CRPT),
   erc20('cslv', 'Coineru Silver', 8, '0x6dc05497f0b087c7692816e6acaa8bdda73907fc', UnderlyingAsset.CSLV),
   erc20('csp', 'Caspian Token', 18, '0xa6446d655a0c34bc4f05042ee88170d056cbaf45', UnderlyingAsset.CSP),
+  erc20('cusd', 'Celo USD', 18, '0x765de816845861e75a25fca122bb6898b8b1282a', UnderlyingAsset.CUSD),
```

Comment:
I think we should make a new network type for CELO-based ERC20 tokens, as these tokens don't exist in the ETH chains. 
Like: https://github.com/BitGo/BitGoJS/blob/11b60b28046f8800f67d02fe235b85c15e449f5d/modules/statics/src/account.ts#L170